apps/src/blockly/addons/blockSvgUnused.js

```diff
@@ -100,7 +28,7 @@ export default class BlockSvgUnused {
       'circle',
       {
         fill: '#7665a0',
```

Comment:
Not part of your work per se, but just a note that I do see this color in shared/css/color.scss as 'purple'. Not sure if we can access the file from here.